tests/ci_tests/code_formatter/crewai/test_crewai_tasks.py

```diff
@@ -0,0 +1,200 @@
+"""Tests for coffee_maker/code_formatter/crewai/tasks.py"""
+
+from unittest import mock
+
+import pytest
+from crewai import Agent, Task
+
+from coffee_maker.code_formatter.crewai.tasks import (
+    EXPLANATIONS_DELIMITER_END,
+    EXPLANATIONS_DELIMITER_START,
+    MODIFIED_CODE_DELIMITER_END,
+    MODIFIED_CODE_DELIMITER_START,
+    create_reformat_task,
+    create_review_task,
+)
+
+
+@pytest.fixture
+def mock_agent():
+    """Create a real Agent instance for testing"""
+    return Agent(role="Test Agent", goal="Test goal", backstory="Test backstory", verbose=False)
+
+
+class TestDelimiters:
+    """Test that delimiter constants are defined"""
+
+    def test_delimiters_exist(self):
+        """Test that all required delimiters are defined"""
```

Comment:
Line 24-28: Added a docstring to the function as specified in the style guide.
```suggestion
    """Deletes a file.\n\n    Args:\n        file_path (str): The path to the file.\n    """
```